packages/augur-sdk/src/state/getter/Liquidity.ts

```diff
@@ -0,0 +1,82 @@
+import { BigNumber } from 'bignumber.js';
+import * as t from 'io-ts';
+import { DB } from '../db/DB';
+import { Getter } from './Router';
+import { Augur } from '../../index';
+import * as _ from 'lodash';
+
+export const Order = t.type({
+    price: t.string,
+    amount: t.string,
+});
+  
+export const OutcomeOrderBook = t.type({
+    bids: t.array(Order),
+    asks: t.array(Order),
+});
+
+
+export const OrderBook = t.dictionary(
+    t.string,
+    OutcomeOrderBook,
+);
+
+export const GetMarketLiquidityRankingParams = t.type({
+    orderBook: OrderBook,
+    numTicks: t.string,
+    marketType: t.number,
+    reportingFeeDivisor: t.string,
+    marketFeeDivisor: t.string,
+    numOutcomes: t.number,
+    spread: t.number,
+});
+
+export interface MarketLiquidityRanking {
+    marketRank: number;
+    totalMarkets: number;
+    hasLiquidity: boolean;
+}
+
+export class Liquidity {
+  static getMarketLiquidityRankingParams = GetMarketLiquidityRankingParams;
+
+  @Getter('getMarketLiquidityRankingParams')
+  static async getMarketLiquidityRanking(augur: Augur, db: DB, params: t.TypeOf<typeof Liquidity.getMarketLiquidityRankingParams>): Promise<MarketLiquidityRanking> {
+    const liquidityScore = await augur.liquidity.getLiquidityForSpread({
+        orderBook: params.orderBook,
+        numTicks: new BigNumber(params.numTicks),
+        marketType: params.marketType,
+        reportingFeeDivisor: new BigNumber(params.reportingFeeDivisor),
+        marketFeeDivisor: new BigNumber(params.marketFeeDivisor),
+        numOutcomes: params.numOutcomes,
+        spread: params.spread,
```

Comment:
Can this be calculated instead of passed in? Since the orderbook is a parameter

Reply:
I don't understand what you mean. The provided spread is a parameter used to evaluate an orderbook's liquidity. It cant be derived

Reply:
This is just a value that has to be specified. `getLiquidityForSpread` returns a different value, depending on the spread that's passed in.

Reply:
I get it. UI knows the default spread that needs to be passed. Gotcha